doc/ChangeLog

```diff
@@ -2,23 +2,22 @@
 Tag name:  ctsm1.0.dev027
 Originator(s):  negins (Negin Sobhani,UCAR/CSEG,303-497-1224)
 Date: Tue Feb 19 12:57:12 MST 2019
-One-line Summary: Non-constant time initialization for soil hydrology types are moved to SoilHydrologyType.F
+One-line Summary: Non-constant time initialization for soil hydrology types are moved to /src/biogeophys/SoilHydrologyType.F90. 
```

Comment:
Simply "SoilHydrologyType.F90" is fine here. I mainly bring this up because this line should be the same as the one-liner in ChangeSum.

Reply:
Thanks! You are correct. I made the change in the online summary ( SoilHydrologyType.F90)  to match ChangeSum.